### PR TITLE
{devel}[gompi/2021a] CWIPI v0.12.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/c/CWIPI/CWIPI-0.12.0-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+_cwipi_bin = ['c_linear_coupling', 'c_surf_coupling_P1P0_P0P1', 'c_surf_coupling_P1P1',
+              'c_surf_coupling_simple_location', 'c_surf_coupling_vector_exchange', 'c_surf_location_triaP2',
+              'c_surf_sequential_coupling', 'c_vol_coupling_P1P1', 'c_vol_coupling_polyhedra_P1P1',
+              'fortran_plane_PiQj', 'fortran_surf_P1P1', 'fortran_surf_PiQj']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in _cwipi_bin] +
+             ['lib/libcwp.a', 'lib/libcwp.%s' % SHLIB_EXT, 'lib/libcwpf.a', 'lib/libcwpf.%s' % SHLIB_EXT],
+    'dirs': ['include']
+}
```
Comment (smoors) [inline easybuild/easyconfigs/c/CWIPI/CWIPI-0.12.0-gompi-2021a.eb:33]: @lexming can you add a `sanity_check_commands`?

Comment (lexming) [reply easybuild/easyconfigs/c/CWIPI/CWIPI-0.12.0-gompi-2021a.eb:33]: In this case it's better to not add them. The tools from CWIPI need at least 2 processes to run, which might not always be the case in the build system and also `%(mpi_cmd_prefix)s` defaults to 1.

Comment (smoors) [reply easybuild/easyconfigs/c/CWIPI/CWIPI-0.12.0-gompi-2021a.eb:33]: ok, can you add that as a comment?

Comment (lexming) [reply easybuild/easyconfigs/c/CWIPI/CWIPI-0.12.0-gompi-2021a.eb:34]: done

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node305.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/8932ee1cc5ddf7faafd0ba2944b558b4 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**